db/db_impl.h

```diff
@@ -396,7 +396,7 @@ class DBImpl : public DB {
   Status TEST_SwitchMemtable(ColumnFamilyData* cfd = nullptr);
 
   // Force current memtable contents to be flushed.
-  Status TEST_FlushMemTable(bool wait = true,
+  Status TEST_FlushMemTable(bool wait = true, bool allow_write_stall = false,
```

Comment:
it might be less error-prone if we add new arguments with default value at the end of the argument list. Then you won't have to change transaction test. But it's not a big deal either way.

Reply:
I think given how rarely this non-default CF is used in tests now and that we found and fixed the only file where it was actually used, I'd rather leave 2 boolean flags first and then the CF pointer, I think.

Updated the diff with the HISTORY.md note and test fixes. Thank you for reviews!